src/executables/util_service.c

```diff
@@ -88,11 +88,13 @@ typedef enum
   ACCESS_CONTROL,
   RESET,
   INFO,
-  SC_COPYLOGDB,
```

Comment:
I'm wondering why some of enum or definition gets out, but some are not. 
If you're intending to minimize the change, I think you can leave this SC_COPYLOGDB/SC_APPLYLOGDB. It seems sufficient to modify us_Command_map.
 And if you're removing all related, you should remove others like COMMAND_TYPE_COPYLOGDB/COMMAND_TYPE_APPLYLOGDB, us_hb_process_copylogdb(), us_hb_process_applylogdb(), us_hb_copylogdb_start/stop(), UTIL_COPYLOGDB, etc..

Reply:
OK, you are right 🥲
I will find and remove them all.